### ISSUE TITLE
can't see a specific product detail page anymore (99 % of my product-urls work without problems)

### ISSUE THREAD

**Issue by yunits** [FriendsOfShopware/FroshPlatformThumbnailProcessor#63]
### PHP Version

8.0.21

### Shopware Version

6.4.14.0

### Actual behaviour

Hi, can't call a specific product url. I see this error the first time.

"Leider ist etwas schief gelaufen"

```
24. August 2022 um 15:56:35
request
CRITICAL
Uncaught PHP Exception TypeError: "explode(): Argument #2 ($string) must be of type string, null given" at /meinPfad/custom/plugins/FroshPlatformThumbnailProcessor/src/Storefront/Framework/Twig/Extension/UrlEncodingTwigFilter.php line 35 {"exception":"[object] (TypeError(code: 0): explode(): Argument #2 ($string) must be of type string, null given at /meinPfad/custom/plugins/FroshPlatformThumbnailProcessor/src/Storefront/Framework/Twig/Extension/UrlEncodingTwigFilter.php:35)"} []
24. August 2022 um 15:56:35
php
CRITICAL
Uncaught Error: explode(): Argument #2 ($string) must be of type string, null given {"exception":"[object] (TypeError(code: 0): explode(): Argument #2 ($string) must be of type string, null given at /meinPfad/custom/plugins/FroshPlatformThumbnailProcessor/src/Storefront/Framework/Twig/Extension/UrlEncodingTwigFilter.php:35)"} []
```

### Expected behaviour

see product

### Steps to Reproduce?

I don't know how to reproduce. This product had no images. But it is not the first product without image, but the first with an error. After assigning an image, the error still occurs.

**Comment by tinect** [comment]
Hey @yunits

If there is no image, the function would not be called. 🤔 

I'll add a check, but would you please verify what data is sent?
`var_dump($mediaUrl);` at `https://github.com/FriendsOfShopware/FroshPlatformThumbnailProcessor/blob/master/src/Storefront/Framework/Twig/Extension/UrlEncodingTwigFilter.php#L27` f.e.

**Comment by tinect** [comment]
And, could you please verify these are working for you?
https://github.com/FriendsOfShopware/FroshPlatformThumbnailProcessor/pull/64/files

**Comment by yunits** [comment]
After looking at the stack trace, I've seen "configurator options". The error occurs if you assign empty media as "configurator option".
![image](https://user-images.githubusercontent.com/59102282/186590979-9374d5e9-9dbe-4aaf-9229-fa3d175df326.png)


Before I've figured that out, I did the following:
I don't know which part is important for your. I switched from "prod" to "dev" and got this error on product page:
![image](https://user-images.githubusercontent.com/59102282/186590367-aac8febc-1d04-40e3-b5a3-e02bada56cb0.png)
![image](https://user-images.githubusercontent.com/59102282/186590672-ada6f506-c21d-4496-a670-979df2376fe2.png)

pull 64 didn't worked for me.
